directory/cli/src/test_user_simple.py

```diff
@@ -0,0 +1,99 @@
+
+from unittest import mock
+from unittest.mock import ANY
+
+import directory
+import ipa_utils
+import test_utils
+import appliance_cli.utils
+from appliance_cli.testing_utils import click_run
+
```

Comment:
Thanks for adding these tests, these look great. A couple of situations which jump out at me as possibly also worth testing (since these are also major flows through the CLI), and which should be able to be handled in similar ways:

- running `user create` where the user does choose to set a UID, to check that this works and the UID is correctly passed through to the `ipa` command;

- running `user modify` when the user attempting to be modified doesn't exist, to check this (likely common) error situation is handled correctly.

Reply:
Did you decide to hold off on adding these for now? :slightly_smiling_face: